src/components/Dashboard/AccountBanner.js

```diff
@@ -205,21 +205,37 @@ const BannerWithProbability = ({ activeBalance }) => {
         </span>
       </span>
       <Help hint="How is the probability calculated?">
-        <p>
-          This is a numerical estimate of your likelihood of being selected for
-          arbitration. It’s calculated by dividing your active ANJ balance
-          against the Court's total active ANJ balance during the current term.
-        </p>
-        <p
-          css={`
-            margin-top: ${1 * GU}px;
-          `}
-        >
-          You currently have &lt;1% of all activated ANJ and hence are unlikely
-          to be drafted unless a dispute goes to the final round or many
-          disputes are created. Activate more ANJ to increase your chances of
-          being selected as a juror.
-        </p>
+        {(() => {
+          if (probablilityTooLow) {
+            return (
+              <>
+                <p>
+                  This is a numerical estimate of your likelihood of being
+                  selected for arbitration. It’s calculated by dividing your
+                  active ANJ balance against the Court's total active ANJ
+                  balance during the current term.
+                </p>
+                <p
+                  css={`
+                    margin-top: ${1 * GU}px;
+                  `}
+                >
+                  You currently have &lt;1% of all activated ANJ and hence are
+                  unlikely to be drafted unless a dispute goes to the final
+                  round or many disputes are created. Activate more ANJ to
+                  increase your chances of being selected as a juror.
+                </p>
+              </>
+            )
+          }
+          return (
```

Comment:
Ah, I wonder if we should just use the same first paragraph in both cases, but only show the second one if you have < 1%?

Reply:
that is not what we have on the  [figma files](https://www.figma.com/file/IdVA6iMn0cWLV5imylwEUP/%E2%9C%85-MASTER%3A-Court-Dashboard?node-id=3450%3A0)

Reply:
Ah, I think that's because it doesn't have the copy adjustments made after the fact.

In the Figma file, both of the initial paragraphs are the same.

Reply:
![Screen Shot 2020-03-05 at 3 16 35 PM](https://user-images.githubusercontent.com/11763623/76012030-5b005100-5ef4-11ea-8686-5c21c0c49be0.png)

not sure which one should i follow because those are differents when you have less than 1% or more than 1

Reply:
Hmm yes, I only saw the left and that one suggested the two had the same first paragraphs.

I think what we have now is clear (I think the copy on the right side is more clear than the left) and let's leave it until more people say they're confused 😅